scene/gui/text_edit.cpp

```diff
@@ -511,7 +511,7 @@ void TextEdit::_notification(int p_what) {
 			_update_scrollbars();
 
 			RID ci = get_canvas_item();
-			RenderingServer::get_singleton()->canvas_item_set_clip(get_canvas_item(), true);
+			RenderingServer::get_singleton()->canvas_item_set_clip(get_canvas_item(), false);
```

Comment:
See https://github.com/godotengine/godot/issues/32996 – this may cause other issues.